packages/blade/src/components/Box/BaseBox/types/index.ts

```diff
@@ -1,3 +1,3 @@
-export type { BaseBoxProps } from './props.types';
+export type { BaseBoxProps, BoxProps, StyledProps } from './props.types';
```

Comment:
I have kept all types in BaseBox/types only. I do re-export it from index for consumers. 

Basically I have kept types according to their usecase (spacing, responsive, props, etc) instead of components. Which is why styled props are also exported from here first.